support/install.sh

```diff
@@ -0,0 +1,232 @@
+		HAS_SYSTEMD="true"
+	fi
+}
+
+install() {
+	local release_name="nt-connect_${VERSION}_${OS}_${ARCH}"
+	local dlpath="/releases/download/${VERSION}/${release_name}.tar.gz"
+	mkdir -p "${INSTALL_DIR}"
+	if test ! -f "${INSTALL_DIR}/${release_name}.tar.gz"; then
+		curl -s -f -L "https://github.com/NorthernTechHQ/nt-connect${dlpath}" \
+			-o "${INSTALL_DIR}/${release_name}.tar.gz"
+	fi
+	if test -d /etc/nt-connect; then
+		tar -xzf "${INSTALL_DIR}/${release_name}.tar.gz" \
+			--exclude ./etc/nt-connect \
+			-C /
+	else
+		tar -xzf "${INSTALL_DIR}/${release_name}.tar.gz" -C /
+	fi
+}
+
+authenticate() {
+	local USERNAME=""
+	local PASSWORD=""
+	printf 'Enter username: ' 1>&2
+	read USERNAME
+	while true; do
+		printf 'Enter password: ' 1>&2
+		stty -echo
+		read PASSWORD
+		stty echo
+
+		code=$(curl -w '%{http_code}' \
+			-f -s \
+			-u "${USERNAME}:${PASSWORD}" \
+			-X POST \
+			-o "${INSTALL_DIR}/authz.jwt" \
+			"${SERVER_URL}/api/management/v1/useradm/auth/login")
+		if test $code -eq 200; then
+			break;
+		elif test $code -eq 401; then
+				printf "Authentication failed\n"
+				continue;
+		else
+				printf "Unknown error attempting to login" 2>&1
+				exit 1;
+		fi
+	done
+	echo ""
+	SESSION_TOKEN=$(cat ${INSTALL_DIR}/authz.jwt)
+	rm -f ${INSTALL_DIR}/authz.jwt
+}
+
+logout() {
+	curl "${SERVER_URL}/api/management/v1/useradm/auth/logout" \
+		-s -X POST \
+		-H "Authorization: Bearer ${SESSION_TOKEN}"
+}
+
+bootstrap() {
+	/usr/bin/nt-connect bootstrap 2>&1 1> /dev/null
+	printf 'Enter server url [https://app.alvaldi.com]: '
+	read SERVER_URL
+	if test -z "${SERVER_URL}"; then
+		SERVER_URL="https://app.alvaldi.com"
+	fi
+	if test -z "$SESSION_TOKEN"; then
+		authenticate
+		trap logout INT QUIT TERM EXIT
+	fi
+	if test -z "$TENANT_TOKEN"; then
+		TENANT_TOKEN=$(curl -s -f "${SERVER_URL}/api/management/v1/tenantadm/user/tenant" \
+			-H "Authorization: Bearer ${SESSION_TOKEN}" | \
+			jq -r .tenant_token)
+	fi
+	umask 066
+
+	# Update the configuration by providing tenant token and server URL.
+	if ! jq ".API.TenantToken=\"${TENANT_TOKEN}\" | .API.ServerURL=\"${SERVER_URL}\"" \
+		/etc/nt-connect/nt-connect.json > \
+		/etc/nt-connect/nt-connect.json.new; then
+		echo "Failed to bootstrap nt-connect configuration";
+		exit 1;
+	fi
+	cp -n /etc/nt-connect/nt-connect.json /etc/nt-connect/nt-connect.json.bak
+	mv /etc/nt-connect/nt-connect.json.new \
+		/etc/nt-connect/nt-connect.json
+
+	# Authorize device
+	cat /var/lib/nt-connect/identity.json | \
+		jq '.identity_data = (.id_data | fromjson) | del(.id_data)' | \
+		curl -s "${SERVER_URL}/api/management/v2/devauth/devices" \
+		-H 'Content-Type: application/json' \
+		-H "Authorization: Bearer ${SESSION_TOKEN}" -d '@-' > /dev/null
+
+	echo "nt-connect installed successfully!"
+
+	# Enable and start systemd service
+	if test "$HAS_SYSTEMD" = "true"; then
+		systemctl enable nt-connect
+		systemctl start nt-connect
+	else
+		echo "To start the daemon, run:"
```

Comment:
A little bit more of a warning would have helped me here:

```suggestion
		echo "Warning: nt-connect is not running - did not find systemctl(!)"
		echo "To start the daemon, run:"
```